images/Dockerfile.matlab

```diff
@@ -1,84 +1,25 @@
 
-RUN apt update \
- && apt install -y software-properties-common \
- && add-apt-repository -y 'ppa:apptainer/ppa' \
- && apt update \
- && apt install -y apptainer-suid \
- && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
-
-RUN apt-get update && apt-get install -y ca-certificates libseccomp2 \
-   uidmap squashfs-tools squashfuse fuse2fs fuse-overlayfs fakeroot \
-   s3fs netbase less parallel tmux screen vim emacs htop curl \
-   git build-essential \
-   && rm -rf /tmp/*
-
-RUN curl --silent --show-error "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" \
-  -o "awscliv2.zip" && unzip awscliv2.zip \
-  && ./aws/install && rm -rf ./aws awscliv2.zip
-
-# Install jupyter server proxy and desktop
-RUN curl -fsSLo /usr/share/keyrings/brave-browser-archive-keyring.gpg https://brave-browser-apt-release.s3.brave.com/brave-browser-archive-keyring.gpg \
-   && echo "deb [signed-by=/usr/share/keyrings/brave-browser-archive-keyring.gpg] https://brave-browser-apt-release.s3.brave.com/ stable main" | sudo tee /etc/apt/sources.list.d/brave-browser-release.list \
-   && apt-get -y update \
-   && apt-get install -y  \
-       dbus-x11 \
-       libgl1-mesa-glx \
-       xfce4 \
-       xfce4-panel \
-       xfce4-session \
-       xfce4-settings \
-       xorg \
-       xubuntu-icon-theme \
-       brave-browser \
-    && rm -rf /tmp/*
-
-# Remove light-locker to prevent screen lock
-ARG TURBOVNC_VERSION=3.0.2
-RUN wget -q "https://sourceforge.net/projects/turbovnc/files/${TURBOVNC_VERSION}/turbovnc_${TURBOVNC_VERSION}_amd64.deb/download" -O turbovnc_${TURBOVNC_VERSION}_amd64.deb && \
-   apt-get install -y -q ./turbovnc_${TURBOVNC_VERSION}_amd64.deb && \
-   apt-get remove -y -q light-locker && \
-   rm ./turbovnc_${TURBOVNC_VERSION}_amd64.deb && \
-   ln -s /opt/TurboVNC/bin/* /usr/local/bin/ \
-    && rm -rf /tmp/*
-
-# apt-get may result in root-owned directories/files under $HOME
-RUN mkdir ${EXTRA_DIR} && chown -R $NB_UID:$NB_GID $HOME ${EXTRA_DIR}
-
-USER $NB_USER
-
-RUN pip install --no-cache-dir jupyter-remote-desktop-proxy jupyterlab_nvdashboard
-
-ENV MAMBA_NO_LOW_SPEED_LIMIT=1
-
-RUN mamba install --yes datalad rclone 'h5py>3.3=mpi*' ipykernel zarr blosc gcc eccodes websockify \
-  && wget --quiet https://raw.githubusercontent.com/DanielDent/git-annex-remote-rclone/v0.7/git-annex-remote-rclone \
-  && chmod +x git-annex-remote-rclone && mv git-annex-remote-rclone /opt/conda/bin \
-  && conda clean --all -f -y && rm -rf /tmp/*
-
-RUN pip install --no-cache-dir plotly jupyter_bokeh jupytext nbgitpuller datalad_container \
+RUN pip install --no-cache-dir datalad 'h5py>3.3' zarr pyopenssl plotly jupyter_bokeh jupytext nbgitpuller datalad_container \
     datalad-osf dandi nibabel nilearn pybids spikeinterface neo \
-    'pydra>=0.25' 'pynwb>=2.8.3' 'nwbwidgets>=0.10.2' hdf5plugin s3fs h5netcdf "xarray[io]"  \
+    'pydra>=0.17' 'pynwb>=2.3.1' 'nwbwidgets>=0.10.2' hdf5plugin s3fs h5netcdf "xarray[io]"  \
```

Comment:
I originally missed this but we should keep this upgrade.  Filed https://github.com/dandi/dandi-hub/pull/227.

Reply:
Thanks for spotting this! Sorry about this, I definitely mistook when I merged the code of the various tests I did.

Reply:
No worries at all